### PR TITLE
BDD

### PR DESCRIPTION
**Pull request by jacopocarlini**
#### List of Changes

Added `integration-test` folder with the BDD tests. 
The tests are written using Gherkin and [Cucumber JS](https://cucumber.io/docs/installation/javascript/). The folder structure used is explained [here](https://medium.com/digio-australia/a-simple-guide-to-getting-started-with-cucumberjs-and-puppeteer-2787c0bf46d9).

How to use
- use `properties.json` to configure your environment
- `yarn test` to launch the features
- Remember to start the project first

 _Use the `README.md` inside the directory for more usage info_

#### Motivation and Context

To add integration tests. 

#### How Has This Been Tested?

with yarn test

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: integartion-test/features/organizations.feature [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Feature: All about Organizations
```
Comment (pasqualespica) [inline integartion-test/features/organizations.feature]: typo folder name `integartion-test`

File: integartion-test/features/organizations.feature [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+  Scenario: An organization creates an enrollment
+    Given the organization "777777"
+    When the organization enrolls in the service "service-1"
+    Then the organization is enrolled in the service
```
Comment (pasqualespica) [inline integartion-test/features/organizations.feature]: you have to figure out whether to check the http status or not.
Ex. `Then the organization is enrolled in the service` in `Then the organization gets the status code 201`

**Review by pasqualespica — APPROVED** [review]
great work 💪 
Could be useful to [publish report](https://smartbear.wistia.com/medias/nrgptu7825) or to [generated it locally](https://www.npmjs.com/package/cucumber-html-reporter)

File: integartion-test/features/organizations.feature (hunk 1 above)
Comment (jacopocarlini) [reply integartion-test/features/organizations.feature]: Outdated

File: integartion-test/features/organizations.feature (hunk 2 above)
Comment (jacopocarlini) [reply integartion-test/features/organizations.feature]: Outdated

**Pull request merged**